content/hi/docs/reference/glossary/cluster.md

```diff
@@ -0,0 +1,19 @@
+---
+title: क्लस्टर (Cluster)
+id: cluster
+date: 2019-06-15
+full_link: 
+short_description: >
+    वर्कर मशीनों का एक सेट, जिसे नोड कहा जाता है, जो कंटेनरीकृत एप्लिकेशन चलाता है। प्रत्येक क्लस्टर में कम से कम एक वर्कर नोड होता है।
```

Comment:
```suggestion
    वर्कर मशीनों का एक सेट, जिसे नोड्स कहा जाता है, जो कंटेनरीकृत एप्लिकेशन चलाता है। प्रत्येक क्लस्टर में कम से कम एक वर्कर नोड होता है।
```

Reply:
I'd still go with the original phrasing.